exercise_21.js

```diff
@@ -0,0 +1,23 @@
+/*
+ Implement the getPercentageValue function
+
+it should accept two arguments:
+
+the base number
+
+the percentage
+
+the function should return the percentage value based on the base number and the percentage
+ */
+
+function getPercentageValue(baseNumber, percentage) {
+    if (baseNumber > 0 ) {
+        return ((percentage / baseNumber) * 100 + " percent");
+    }
+    return "Please provide correct number";
+}
+
+console.log(getPercentageValue(20, 5));
+console.log(getPercentageValue(100, 50));
+console.log(getPercentageValue(10, 1));
+console.log(getPercentageValue(150, 100));
```

Comment:
moim zdaniem to NIE jest poprawna logika bo powinno to zwracać liczbę (tak ja rozumiem polecenie) np 
```
console.log(getPercentageValue(20, 10)); // zwraca "50 percent" a powinno zwrocic 2 bo 2 to 10% z liczby 20 
```